.github/workflows/main.yml

```diff
@@ -55,8 +60,31 @@ jobs:
           python manage.py restore_postgresql_database
           # run indexing indexing
           python manage.py get_index_data_from_database -b False
-          # run tests
-          python -m unittest discover -s unit_tests
+          # set up the seond database
+          rm app_data/omero_db_searchengine.zip
+          rm app_data/omero.pgdump
+          wget https://github.com/khaledk2/ice-archh-64/releases/download/new_re_db/omero_train.zip -P app_data
+          unzip app_data/omero_train.zip -d app_data/
+          rm app_data/omero_train.zip
+          python manage.py set_database_configuration -u localhost -s  ${{ job.services.postgres.ports[5432] }} -n postgress -p passwprd -w omero_train -d omero_train -b omero_train.pgdump
+          echo $PGPASSWORD
+          psql -h localhost -U postgress -p ${{ job.services.postgres.ports[5432] }} -c 'CREATE DATABASE omero_train'
+          python manage.py restore_postgresql_database -s omero_train
+          python manage.py get_index_data_from_database -b False -d omero_train
+          rm app_data/omero_train.pgdump
+          # set csv data source
+          python manage.py set_data_source_files -n test_csv
+          # download images csv file
+          wget https://github.com/khaledk2/ice-archh-64/releases/download/test_csv/test_images.zip -P app_data
```

Comment:
same.

Reply:
Should move them to  https://downloads.openmicroscopy.org/images?

Reply:
We need to create a new location e.g. resources